src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -180,6 +209,260 @@ In the example above, the files required are placed within the `/tmp` directory.
 
 Refer to the set of `conduct` commands described in [[Deploying bundles|DeployingBundlesOps]] for more details.
 
+
+## Creating and deploying custom HAProxy configuration for Sandbox
+
+Follow these steps to provide custom HAProxy configuration to ConductR HAProxy running in the Sandbox. To provide a custom configuration for a non-sandbox environment go to the chapter [Creating and deploying custom HAProxy configuration](#creating-and-deploying-custom-haproxy-configuration).
```

Comment:
"Follow these steps to provide custom HAProxy configuration" -> "Follow these steps to provide _a_ custom HAProxy configuration"